src/NuGet.Clients/NuGet.VisualStudio/SolutionRestoreManager/IVsNuGetPackageReferenceProjectUpdateEvents.cs

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+
+namespace NuGet.SolutionRestoreManager
+{
+    /// <summary>
+    /// NuGet PackageReference project update events.
+    /// This API provides means of tracking project updates performance by NuGet on PackageReference projects, in particular updates to the assets file and nuget generated props/targets.
+    /// All events are fired from a threadpool thread.
+    /// </summary>
+    [ComImport]
```

Comment:
Considering that VS2022 discourages embedded interop types and we stopped shipping a props/targets file that enabled embedded interop types, perhaps we should stop adding `[ComImport]` to our interfaces. It's probably a bug that I didn't remove these `[ComImport]` attributes from all the interfaces when I stopped the package from using embed interop.

Reply:
You're right, I don't think we need. 

I'll remove this.